ext/oci8/oci8.stub.php

```diff
@@ -2,534 +2,468 @@
+ * @param resource $statement
  * @alias oci_bind_by_name
  * @deprecated
  */
-function ocibindbyname($statement_resource, string $column_name, mixed &$variable, int $maximum_length = -1, int $type = 0): bool {}
+function ocibindbyname($statement, string $param, mixed &$var, int $max_length = -1, int $type = 0): bool {}
 
-/** @param resource $statement_resource */
-function oci_bind_array_by_name($statement_resource, string $column_name, mixed &$variable, int $maximum_array_length, int $maximum_item_length = -1, int $type = SQLT_AFC): bool {}
+/** @param resource $statement */
+function oci_bind_array_by_name($statement, string $param, mixed &$var, int $max_array_length, int $max_item_length = -1, int $type = SQLT_AFC): bool {}
 
-function oci_free_descriptor(OCILob $lob_descriptor): bool {}
+function oci_free_descriptor(OCILob $lob): bool {}
 
 /**
  * @alias oci_free_descriptor
  * @deprecated
  */
-function ocifreedesc(OCILob $lob_descriptor): bool {}
+function ocifreedesc(OCILob $lob): bool {}
 
-function oci_lob_save(OCILob $lob_descriptor, string $data, int $offset = 0): bool {}
+function oci_lob_save(OCILob $lob, string $data, int $offset = 0): bool {}
 
 /**
  * @alias oci_lob_save
  * @deprecated
  */
-function ocisavelob(OCILob $lob_descriptor, string $data, int $offset = 0): bool {}
+function ocisavelob(OCILob $lob, string $data, int $offset = 0): bool {}
 
-function oci_lob_import(OCILob $lob_descriptor, string $filename): bool {}
+function oci_lob_import(OCILob $lob, string $filename): bool {}
 
 /**
  * @alias oci_lob_import
  * @deprecated
  */
-function ocisavelobfile(OCILob $lob_descriptor, string $filename): bool {}
+function ocisavelobfile(OCILob $lob, string $filename): bool {}
 
-function oci_lob_load(OCILob $lob_descriptor): string|false {}
+function oci_lob_load(OCILob $lob): string|false {}
 
 /**
  * @alias oci_lob_load
  * @deprecated
  */
-function ociloadlob(OCILob $lob_descriptor): string|false {}
+function ociloadlob(OCILob $lob): string|false {}
 
-function oci_lob_read(OCILob $lob_descriptor, int $length): string|false {}
+function oci_lob_read(OCILob $lob, int $length): string|false {}
 
-function oci_lob_eof(OCILob $lob_descriptor): bool {}
+function oci_lob_eof(OCILob $lob): bool {}
 
-function oci_lob_tell(OCILob $lob_descriptor): int|false {}
+function oci_lob_tell(OCILob $lob): int|false {}
 
-function oci_lob_rewind(OCILob $lob_descriptor): bool {}
+function oci_lob_rewind(OCILob $lob): bool {}
 
-function oci_lob_seek(OCILob $lob_descriptor, int $offset, int $whence = OCI_SEEK_SET): bool {}
+function oci_lob_seek(OCILob $lob, int $offset, int $whence = OCI_SEEK_SET): bool {}
 
-function oci_lob_size(OCILob $lob_descriptor): int|false {}
+function oci_lob_size(OCILob $lob): int|false {}
 
-function oci_lob_write(OCILob $lob_descriptor, string $string, ?int $length = null): int|false {}
+function oci_lob_write(OCILob $lob, string $data, ?int $max_length = null): int|false {}
```

Comment:
Based on discussion elsewhere, we'll probably want to keep this at `$length` -- though we can change that later.